BunnyHopMaster/Assets/Scripts/Player/PlayerProgress.cs

```diff
@@ -34,19 +31,14 @@ private void Update()
             return;
         }
 
-        if (Input.GetKeyDown(KeyCode.R))
-        {
-            Respawn();
-        }
-
-        if (gameObject.transform.position.y < playerFallingBoundsReset)
+        if (Input.GetKeyDown(HotKeyManager.Instance.GetKeyFor(PlayerConstants.ResetLevel)))
         {
             Respawn();
         }
 
         if (currentCheckpoint != null && !GameManager.Instance.didWinCurrentLevel)
         {
-            if (currentCheckpoint.level == GameManager.GetCurrentLevel().numberOfCheckpoints)
+            if (currentCheckpoint.level == GameManager.GetCurrentLevel().numberOfCheckpoints || Input.GetKeyDown(HotKeyManager.Instance.GetKeyFor(PlayerConstants.WinMenu)))
```

Comment:
i don't think we want this. This literally lets them hit a button and win the level. what are you trying to do here?